web-app/server/compileIR.js

```diff
@@ -0,0 +1,79 @@
+var fs = require('fs');
+var exec = require('child_process').exec;
+var LLFI_BUILD_ROOT = require('./utils/config').LLFI_BUILD_ROOT;
+var execPromise = require('./utils/execPromise').execPromise;
+
+exports.processCompileIR = function (req, res) {
```

Comment:
@abrahamchan  I have fixed the typo and renamed compileIR to compileToIR in all references.